src/main/java/geekbrains/java2/ht2/Move.java

```diff
@@ -0,0 +1,109 @@
+package geekbrains.java2.ht2;
+
+import java.util.Random;
+import java.util.Scanner;
+
+class Move {
```

Comment:
\[Checkstyle\] ERROR: Utility classes should not have a public or default constructor\.